### DISCUSSION TITLE
field validation support regular expression not supported in nested validation rule ?

### DISCUSSION THREAD

**Discussion by kstang** [directus/directus#14250] [Archive]
seems I cannot find the _regex when the validation rule nested with 'And' , please see image below 
![Screenshot 2022-07-02 at 3 22 43 AM](https://user-images.githubusercontent.com/8650432/176957422-6a862c47-e647-411a-ab18-aae8e70191f2.png)

but the selection of 'Matches RegExp' found  when the validation not nested :
![Screenshot 2022-07-02 at 3 25 20 AM](https://user-images.githubusercontent.com/8650432/176991012-e4a8ff5e-0469-4d8e-acd7-e099686f859c.png)

**Comment by rijkvanzanten** [comment]
I think you've uploaded the same image twice

**Reply by kstang** [reply]
yes, updated the image again .

**Reply by kstang** [reply]
seems the validation will be done on changed field only. 

so I cannot have the rule like 

 (contact_typer = 'mobile' ) and number start_with 9
or 
(contact_type = 'home' ) and  number start_with 2
 
as contact type do not change each time

**Reply by rijkvanzanten** [reply]
That's correct, though something we're aiming to improve very soon

**Reply by kstang** [reply]
ok ic. noted with thx .
